cypress-tests/cypress/e2e/admin/pageBuilder/category/pageCategoriesCrud.cy.ts

```diff
@@ -0,0 +1,92 @@
+import { customAlphabet } from "nanoid";
+
+context("Page Builder - Category CRUD", () => {
+    const nanoid = customAlphabet("abcdefghijklmnopqrstuvwxyz");
+    const category_name = nanoid(6);
+    const category_name_edited = nanoid(6);
+    const category_slug = nanoid(6);
+    const category_url = "/" + nanoid(6) + "/";
+    const category_url_edited = "/" + nanoid(6) + "/";
+
+    beforeEach(() => {
+        cy.login();
+        cy.pbDeleteAllCategories();
+    });
+
+    it("Should be able to create, edit, and immediately delete a category", () => {
+        cy.visit("/page-builder/categories");
+
+        // Creates new category.
+        cy.findByTestId("data-list-new-record-button").click();
+        cy.findByTestId("pb.category.new.form.name").type(category_name);
+        cy.findByTestId("pb.category.new.form.slug").type(category_slug);
```

Comment:
BTW I just realised we've removed some of the form inputs validation steps we had in the initial JS version of the test. Let's have those here as well.